iceberg/iceberg-handler/src/test/results/positive/col_stats.q.out

```diff
@@ -96,7 +96,7 @@ avg_col_len         	3.4444444444444446
 max_col_len         	5                   
 num_trues           	                    
 num_falses          	                    
-bit_vector          	HL                  
+bit_vector          	                    
```

Comment:
why is this change?

Reply:
I suspect it is removed here because of: 
[HIVE-23846](https://issues.apache.org/jira/browse/HIVE-23846)
Avoid unnecessary serialization and deserialization of bitvectors

> In the method getNdvEstimator of <datatype>ColumnStatsDataInspector, it will call isSetBitVectors(), in which it serializes the bitvectors again when we already have deserialized bitvectors ndvEstimator.  

ps: HIVE-17286 introduced HL to avoid expensive string serialisation for ndv

Reply:
I still don't get, how adding minor delta stats merge caused that behavior change? @kasakrisz any hints?

Reply:
````
The bit vector that contains the ndv values is being overwritten by updatecolstats which is called by alter table command.
That's what I want to fix in the alter Table PR
Since bit vector is being overwritten, Hyper log log is not being used
So HL is missing
````

Reply:
A alter table command is run during the doCommit Stage, which seems to unnecessarily overwrite the bit_vector . This should be fixed for iceberg tables. I have created follow up jira to address it.
 https://issues.apache.org/jira/browse/HIVE-27528

Reply:
@simhadri-g, please add a link to a follow-up ticket

Reply:
https://issues.apache.org/jira/browse/HIVE-27528